rhel8/templates/csv/sysctl_values.csv

```diff
@@ -38,3 +38,4 @@ net.ipv6.conf.all.disable_ipv6,1
 net.ipv6.conf.default.accept_source_route,
 net.ipv6.conf.all.accept_source_route,
 net.ipv6.conf.all.forwarding,
+user.max_user_namespaces,1
```

Comment:
Correct value is `0`, https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/rule.yml.

Plus, can you leave this one commented out for now? This rule is optional.

Reply:
I have thought that the idea is that any rule included in the benchmark should have a check, and if it is not needed for e.g. OSPP profile, it simply won't be selected. Is this incorrect?

Reply:
The optional rules are to be in the profile, but without check or remediation. So that compliance to is not enforced.

Reply:
This can be approached in different way, by using `refine-rule` to make changes in a rules `role` within a profile.